mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -693,6 +700,14 @@ linux_node_config {
 }
 ```
 
+The `network_config` block supports:
+
+* `create_pod_range` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) Size of the disk attached to each node, specified
+    in GB. The smallest allowed disk size is 10GB. Defaults to 100GB.
+
+* `disk_type` - (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) Type of the disk attached to each node
```

Comment:
Incorrect field here? The other fields are `pod_ipv4_cidr_block` and `pod_range`

Reply:
Fixed in the docs and in the schema